src/Hl7.Fhir.Base/ElementModel/Types/DateTime.cs

```diff
@@ -150,50 +150,63 @@ private static bool tryParse(string representation, out DateTime value)
             return success;
         }
 
+        public static DateTime operator -(DateTime dateTimeValue, Quantity addValue)
```

Comment:
I would suggest naming the quantity here as `subtractValue` instead of `addValue`.